R/helper_lm_robust_fit.R

```diff
@@ -157,8 +157,7 @@ lm_robust_fit <- function(y,
     return_list <- list(
       coefficients = setNames(as.vector(fit$beta_hat), variable_names),
       se = NA,
-      df = NA,
-      stringsAsFactors = FALSE
+      df = NA
```

Comment:
Do you also need to cut stringsasfactors if(multivariate) is true?

Reply:
This used to be a df, now it's a list. So there's no case where I need that as an item in the list, right?